### ISSUE TITLE
Accessibility Report: Skiplinks Component

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#326]
Use the AMP tool to scan the [Skip Links](http://decanter.stanford.edu/section-components.html#kssref-components-skiplinks) component for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283423

No violations found.

**Comment by sherakama** [comment]
Chrome Audit: 100%

![Screen Shot 2019-03-13 at 8 51 54 PM](https://user-images.githubusercontent.com/550602/54330139-dc6b1f00-45d1-11e9-9993-c81f1ddf47ff.png)
